### PR TITLE
Added changelog entry for read_* changes

### PR DESCRIPTION
**Pull request by Lucretiel**
- These changes were added in #11

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
+# master
+- Reimplemented `Console::read_single_input_event` and `Console::read_console_input` to be more efficient, safe, and correct
+- `Console::read_console_input` no longer returns a `u32`; the numbr of events is the length of the returned vector.
```
Comment (TimonPost) [inline CHANGELOG.md]:
```suggestion
- Make `Console::read_console_input` no return an  `u32`; the number of events is the length of the returned vector.
```

Comment (Lucretiel) [reply CHANGELOG.md]: Should this read "not return an"?

Comment (TimonPost) [reply CHANGELOG.md]: yes

Comment (TimonPost) [inline CHANGELOG.md]:
```suggestion
- Make `Console::read_console_input` return an  `u32`; the number of events is the length of the returned vector.

```

Comment (Lucretiel) [reply CHANGELOG.md]: To be clear the change was to remove the `u32` from read_console_input

**Comment by Lucretiel** [comment]
Fixed merge conflicts

File: CHANGELOG.md (hunk 1 above)
Comment (TimonPost) [reply CHANGELOG.md]: O wait, my bad again. Facepalm. I was in a hurry.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 especially data struct.
 - Significantly updated the `input` structs, so that winapi native types are no longer exposed to the library by crossterm_winapi structs.
 - Removed PartialOrd from types where it didn't really make sense
+- Reimplemented `Console::read_single_input_event` and `Console::read_console_input` to be more efficient, safe, and correct
+- `Console::read_console_input` no longer returns a `u32`; the numbr of events is the length of the returned vector.
```
Comment (TimonPost) [inline CHANGELOG.md]:
```suggestion
- Make `Console::read_console_input` no to return an  `u32`; the number of events is the length of the returned vector.
```

Final comment haha. The idea is just to append 'Make' in order to make it more consistent with the other readme's.

**Comment by TimonPost** [comment]
I will put the release on the schedule for tomorrow morning (in 10 hours).

File: CHANGELOG.md (hunk 2 above)
Comment (Lucretiel) [reply CHANGELOG.md]: Another typo, but fixed :)

**Comment by Lucretiel** [comment]
Anything else needed to merge this?

**Comment by TimonPost** [comment]
Nooe

**Pull request merged**